tasks/db_deploy/orca_sql.py

```diff
@@ -0,0 +1,581 @@
+"""
+Name: orca_sql.py
+
+Description: All of the SQL used for creating and migrating the ORCA schema.
+"""
+# Imports
+from sqlalchemy import text
+
+
+# ----------------------------------------------------------------------------
+# ORCA SQL used for creating the Database
+# ----------------------------------------------------------------------------
+def app_database_sql():
```

Comment:
Does "-> sqlalchemy.text" work as a type hint for the return value?

Reply:
yes, I updated all of the function returns with ` -> text ` as type hints.